pkg/types/installconfig.go

```diff
@@ -85,6 +92,13 @@ type InstallConfig struct {
 	// +optional
 	AdditionalTrustBundle string `json:"additionalTrustBundle,omitempty"`
 
+	// AdditionalTrustBundlePolicy defines conditional policy to add AdditionalTrustBundle
+	// to the nodes' trusted certificate store. If omitted, "ProxyOnly" is considered.
+	// The field can be set to following specified values.
+	// "ProxyOnly" : uses AdditionalTrustBundle when http/https proxy is configured.
+	// "Always" : ignores all conditions and uses AdditionalTrustBundle.
```

Comment:
```suggestion
	// AdditionalTrustBundlePolicy determines when to add the AdditionalTrustBundle
	// to the nodes' trusted certificate store. "ProxyOnly" is the default.
	// The field can be set to following specified values.
	// "ProxyOnly" : adds the AdditionalTrustBundle to nodes when http/https proxy is configured.
	// "Always" : always adds AdditionalTrustBundle.
```

Suggesting some updated wording. 

Also, we are saying these are adds to the "nodes." Is that correct? I'm not sure how this works exactly. I thought it was added as a cluster object.

Reply:
"Adds to nodes" is to say how the bundle is used. The configuration is added to cluster object.